inc/functions.php

```diff
@@ -2841,11 +2841,6 @@ function build_forum_jump($pid="0", $selitem="", $addselect="1", $depth="", $sho
 
 	$pid = (int)$pid;
 
-	if($permissions)
```

Comment:
The `$permissions` argument should be updated to `null` and the description updated to make it clear the passed value is never used anyways.